src/pip/_internal/wheel.py

```diff
@@ -620,6 +620,17 @@ def supported(self, tags=None):
         return bool(set(tags).intersection(self.file_tags))
 
 
+def _is_like_egg_info(
```

Comment:
Should this better be named `contains_egg_info`, since the regex can be matching on any substring?